### DISCUSSION TITLE
0.16.3更新后，系统整体感觉变卡顿

### DISCUSSION THREAD

**Discussion by blanktea2024** [rime/home#1680] [General]
Win11系统，使用16.2后，系统卡的不行，更新的16.3，有明显的好转，但依然有卡，文件管理器相关操作都需要等待一会，不知道各位大佬有没有这方面的问题，可有解决方法

**Comment by oxauwae** [comment]
确有此事，时不时就崩溃。

**Reply by Arvin-Hugh** [reply]
对对对，我觉得是他的日志写入的问题，关键是我还不会关闭他的日志

**Comment by ray19841984** [comment]
請問有解決方法了嗎？目前用一用偶而windows(10)整個系統就會卡住，
要過好幾秒才會動作，移除小狼毫就正常，但這輸入法真的不錯呀~~~

**Comment by blanktea2024** [comment]
使用了一段时间后发现，点击重新布置功能后，卡顿就容易出现。重启算法也不行。

**Comment by owovouo** [comment]
我目前勾了一個選項後，倒是沒有發生工作列卡住
或許有遇到的人可以試試，看看有沒有改善
win11下，時間與語言>輸入>進階鍵盤設定> (勾選) 讓我針對每個應用程式視窗使用不同的輸入法
